src/terminal/cli/init.lua

```diff
@@ -1,5 +1,6 @@
 --- CLI component module.
--- Provides utilities to handle the cursor in terminals.
+-- Provide CLI widgets for terminal applications
+-- NOTE: you MUST `terminal.initialize()` before calling these widgets' `:run()`
```

Comment:
the docs render as markdown. so the line with `NOTE: ....` will be appended after `applications`. An empty line should be inserted to make it render properly.

`make docs` will rebuild the documentation (just make sure to NOT commit the rendered docs, use `make clean` to revert)